src/app.ts

```diff
@@ -356,6 +359,58 @@ const handleIncomingTwilioMessage = async (
       logger.info(
         `SERVER.handleIncomingTwilioMessage (${userId}): Voter has previously confirmed the disclaimer, or one is not required for this organization.`
       );
+
```

Comment:
I intended `handleIncomingTwilioMessage` to be high-level logic for which response strategy a voter should receive (e.g. handleClearedVoter, clarifyHelplineRequest), and for app.ts to handle only high-level logic or logic that's commonly needed across each of these strategies (e.g. `populateIncomingDbMessageTwilioEntry`).

To keep this design pattern and also prevent `handleIncomingTwilioMessage` from getting too much bigger, can you find a better place to put this logic?

Reply:
Okay, I pushed a commit that moves this block and the large one below into helpers in router.ts.  The calls still need to stay here in app.ts because of the high-level behavior: (1) always log, and (2) reply to voted if we aren't cleared yet.